src/persistence/MCWSPersistenceProvider.js

```diff
@@ -29,6 +29,15 @@ export default class MCWSPersistenceProvider extends BaseMCWSPersistenceProvider
     } catch (error) {
       console.warn('MCWSPersistenceProvider:get', error);
 
+      // it's a network error, we don't want to create a new object
+      if (error.status !== 404) {
+        return {
```

Comment:
wouldn't we want some sort of notification to the user, and wouldn't we want to try again until we get it? i don't think we can really do much without the tree loading properly, can we?

Reply:
I mean, in this case, the network is down, so not much we can do, but I think some sort of retry setup could work? I'll look into it.